integration-tests/src/test/java/oracle/kubernetes/operator/ITPodsRestart.java

```diff
@@ -0,0 +1,212 @@
+// Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.util.Map;
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+/**
+ * Simple JUnit test file used for testing Operator.
+ *
+ * <p>This test is used for testing pods being restarted by some properties change.
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ITPodsRestart extends BaseTest {
+
+  private static Domain domain = null;
+  private static Operator operator1;
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes. Create Operator1 and domainOnPVUsingWLST
+   * with admin server and 1 managed server if they are not running
+   *
+   * @throws Exception
+   */
+  @BeforeClass
+  public static void staticPrepare() throws Exception {
+    // initialize test properties and create the directories
+    if (!QUICKTEST) {
+      initialize(APP_PROPS_FILE);
+
+      logger.info("Checking if operator1 and domain are running, if not creating");
+      if (operator1 == null) {
+        operator1 = TestUtils.createOperator(OPERATOR1_YAML);
+      }
+
+      domain = createPodsRestartdomain();
+      Assert.assertNotNull(domain);
+    }
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories
+   *
+   * @throws Exception
+   */
+  @AfterClass
+  public static void staticUnPrepare() throws Exception {
+    if (!QUICKTEST) {
+      logger.info("+++++++++++++++++++++++++++++++++---------------------------------+");
+      logger.info("BEGIN");
+      logger.info("Run once, release cluster lease");
+
+      destroyPodsRestartdomain();
+      tearDown();
+
+      logger.info("SUCCESS");
+    }
+  }
+
+  /**
+   * The property tested is: env: "-Dweblogic.StdoutDebugEnabled=false"-->
+   * "-Dweblogic.StdoutDebugEnabled=true"
+   *
+   * @throws Exception
```

Comment:
The Javadoc should describe the usecase more detail as follows  ...

  Modify the Domain Scoped env property in the Domain Object using  kubectl apply -f domain.yaml 
  Make sure all the Server Pods in the domain got re-started 
  Reference https://github.com/oracle/weblogic-kubernetes-operator/blob/master/site/server-lifecycle.md

Reply:
Done